File: URI/1149.py
total = 0

while True:
	a, n = input().split(' ')
	a = int(a)
	n = int(n)

	if(n <= 0):
		n = input().split(' ')
		n = int(n)
	else:
		for i in range(a, n):
			total += (a + i)
		break

print(total)

File: URI/1149.c
#include <stdio.h>

int main() {
	int a, n;
	scanf("%d", &a);

	while(1) {
		if(n <= 0)
			scanf("%d", &n);
		else
			break;
	}

	int total = 0;
	for(int i = 0; i < n-1; i++) {
		total += (a + i);
	}

	printf("%d", total);

	return 0;
}
